api/tests/integration/infrastructure/repositories/certification-center-repository_test.js

```diff
@@ -307,6 +330,21 @@ describe('Integration | Repository | Certification Center', function () {
       expect(savedCertificationCenter.id).to.exist;
       expect(savedCertificationCenter.name).to.equal(certificationCenter.name);
     });
+
+    it('should update the given certification center with the proper update date', async function () {
```

Comment:
suggestion mettre updateD date `should update the given certification center with the proper updated date`

Reply:
merci !